Workbooks/UpdateCompliance/UpdateComplianceHub/Update Compliance Hub.workbook

```diff
@@ -0,0 +1,1660 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{mappedSubscription}"
+        ],
+        "parameters": [
+          {
+            "id": "67dcb2cb-14b1-4d77-b52b-f728abbf43a2",
+            "version": "KqlParameterItem/1.0",
+            "name": "mappedTenantId",
+            "type": 1,
+            "isGlobal": true,
+            "query": "resources\r\n| take 1\r\n| project tenantId",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "7a2bb320-fa9c-45c8-bbf6-942c8b3a4b0e",
+            "version": "KqlParameterItem/1.0",
+            "name": "mappedSubscription",
+            "label": "subscription",
+            "type": 6,
+            "isGlobal": true,
+            "value": "community-workbooks/UpdateCompliance/Update Compliance Hub",
```

Comment:
can probably delete this value?

Reply:
This is actually needed for now while we wait for API integration.